### ISSUE TITLE
Getting RangeError when running on CI

### ISSUE THREAD

**Issue by johanbrook** [practicalmeteor/meteor-mocha#17]
I'm getting this when running on CircleCI:

```
=> App running at: http://localhost:4096/
spacejam: meteor is ready
spacejam: spawning phantomjs
phantomjs: Running tests at http://localhost:4096/local using test-in-console
phantomjs: RangeError: Maximum call stack size exceeded.
    http://localhost:4096/packages/ecmascript-runtime.js?d552d50ef9d63d143f8f549245241748a4796cec: 182
    http://localhost:4096/packages/practicalmeteor_chai.js?1bddcb966ae831c8acf43bd61b0e14a06d45b47b: 3833
spacejam: phantomjs exited with code: 6
spacejam: killing meteor
spacejam: meteor killed with signal: SIGTERM
spacejam: Unhandled error in meteor client side code. Exiting.
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0

npm test returned exit code 1
```

I'm using SpaceJam, and here's the test script:

```
spacejam test-packages --driver-package=practicalmeteor:mocha-console-runner ./
```

This works good locally, so I find it really weird it doesn't run well on CI.

**Comment by jsep** [comment]
Thanks @johanbrook  for testing our package.

Try the following: 
1. Add the release option to spacejam. `spacejam --release 1.3-rc.1 ....` 
2. Specify in our package `api.versionsFrom('1.3-rc.1)`.

This will force meteor to run with the latest versions of the packages.

**Comment by johanbrook** [comment]
So basically running Meteor 1.3 when testing? I.e. is this issue fixed in 1.3?

**Comment by jsep** [comment]
Oh sorry @johanbrook since we are working with the integration for Meteor 1.3 I thougth it was 1.3 specific, my bad :sweat_smile:.

Can you provide code examples? Also we run into with a similar issue before and its fixed in the latest package and spacejam versions. 

Please add `practicalmeteor:mocha` with constraint version in your `Package.onTest`.
For 1.2 and before use: `api.use('practicalmeteor:mocha@2.1.0_7')`
For 1.3-rc.1 use:  `api.use('practicalmeteor:mocha@2.1.1-rc.1'')`

**Comment by johanbrook** [comment]
I tried using `2.1.0_7` now and pushed to Circle (https://circleci.com/gh/lookback/meteor-dropdowns/45), but same error. The repo is here: https://github.com/lookback/meteor-dropdowns. CircleCI *should* use latest SpaceJam module from npm, so I don't understand what's different there. I noticed now that I run PhantomJS 2.0 locally though, might be different on Circle.

It's not a biggie: if it'll be fixed in 1.3, that's okay.

**Comment by jsep** [comment]
@johanbrook that's the problem, spacejam use phanthom 1.9, this version have problems with `Object.defineProperty` which its used by `chai.should` to add the `should` property to every object.

In phantom 2.1 this issue it's fixed, until we update to phantom 2.1 you can't use should.

**Comment by johanbrook** [comment]
Ah, gotcha. Thanks for clarifying.

Is Phantom 2.1 fairly stable these days? Or, when will 2.1 be required of this package?

**Comment by rbabayoff** [comment]
@johanbrook you can the latest version of phantomjs on your system, and then just run spacejam with --use-system-phantomjs. This will solve your problem - let us know it works so we can close this. We're about to also update the internal phantomjs in spacejam soon. See this issue for more info:

https://github.com/practicalmeteor/spacejam/issues/23#issuecomment-174550720

**Comment by johanbrook** [comment]
Ah, gotcha. Will try and get back to you.

**Comment by jsep** [comment]
Closing this - resolved in spacejam 1.6.0 and this belongs to spacejam's repo anyway and is a duplicate of https://github.com/practicalmeteor/spacejam/issues/23.
